LFS-861: The CARDS app name should not default to LFS

marta- opened the pull request:
Also added new LFS logo and new temporary CARDS logo

marta- committed:
LFS-861: The CARDS app name should not default to LFS
Also added new LFS logo and new temporary CARDS logo

sashaandjic commented:
LFSl logo shown for cards$Kids and cards4CaRe

Default
![image](https://user-images.githubusercontent.com/23243244/105751316-d5263380-5f13-11eb-9adb-38fe2679e6cd.png)

LFS mode
![image](https://user-images.githubusercontent.com/23243244/105751475-0999ef80-5f14-11eb-8863-57ed723cf968.png)

Cards for CaRe
![image](https://user-images.githubusercontent.com/23243244/105751876-95ac1700-5f14-11eb-9482-68a6c554096f.png)

Cards for Kids
![image](https://user-images.githubusercontent.com/23243244/105752294-1834d680-5f15-11eb-8ad1-980906ba5eec.png)

sashaandjic reviewed (approved):
tested

marta- committed:
Merge branch 'dev' into LFS-861

marta- committed:
LFS-861: The CARDS app name should not default to LFS
Removed app name and theme color from pom and specified them in config.json only

marta- committed:
LFS-861: The CARDS app name should not default to LFS
Swapped the components of the browser page title (app name last) to make titles of apps with long names more readable

marta- committed:
Merge branch 'LFS-861' of https://github.com/ccmbioinfo/lfs into LFS-861

marta- committed:
The CARDS app name should not default to LFS
Fixed dev merge artefacts

marta- committed:
LFS-861: The CARDS app name should not default to LFS
Fixed more dev merge artefacts

IntegralProgrammer committed:
LFS-861: The CARDS app name should not default to LFS

Add the /libs/lfs/resources/ objects for the lfs-resources-clinical-data bundle

sashaandjic reviewed (approved):
tested: proper name is appearing in the login dialog, correct logo,color and options are shown on the dashboard
